### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Kury Dorian and Buxtorf Basile

### DISCUSSION THREAD

**Discussion by nairod22** [heig-vd-dai-course/heig-vd-dai-course#480] [Show and tell]
We want to do a manager that helps tracking a project progress. We'll initially implement basic functionalities including create a project, add some tasks and mark them as finished/resolved. We could later add user hierarchy and login if possible. We use TCP to guarantee data integrity.

**Comment by ludelafo** [comment]
Looks good to us!

**Comment by BasileBux** [comment]
Project done: 

Link to the repo: https://github.com/nairod22/ProjectManager
Commit hash: e48b04d2914661d006db272e1d42a4ebe896da16
